packages/styles-primeng-workspace/projects/styles-primeng/src/lib/primeng-theme/theme-base/_components.scss

```diff
@@ -0,0 +1,106 @@
+@import './mixins';
+@import './colors';
+
+@import './components/input/editor';
+
+@layer primeng {
+  @import './common';
+
+  //Input
+  @import './components/input/autocomplete';
+  @import './components/input/calendar';
+  @import './components/input/cascadeselect';
+  @import './components/input/checkbox';
+  @import './components/input/chips';
+  @import './components/input/colorpicker';
+  @import './components/input/dropdown';
```

Comment:
Note: Seeing all those inputs, I'm wondering how big is the output

Reply:
![image](https://github.com/swisspost/design-system/assets/33580481/bc1d6fc1-1f68-48e7-89f6-28b3ff8e6255)

The output size is 164KB, it is equivalent to PrimeNg's built-in Bootstrap theme:
![image](https://github.com/swisspost/design-system/assets/33580481/1f178bb3-55a8-45b0-ad73-b8b778b76dad)